Engine/include/math/VectorMath.hpp

```diff
@@ -0,0 +1,189 @@
+//
+// VectorMath.hpp
+// VGD Engine
+//
+// Created by Benjamin Arnold on 29 Aug 2015
+//
+
+/*! \file VectorMath.hpp
+* @brief defines common vector math operations.
+*/
+
+#pragma once
+
+namespace vgd {
```

Comment:
We should make the name more unique so it will be easier to find and replace later on.